pkgs/by-name/nn/nn/package.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, lib, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "nn";
+  version = "1.85.0";
```

Comment:
1.85.0 is the "last official release", released in 2015, 9 years ago.

Looks like there is a [recent release](https://github.com/sakov/nn-c/commit/8e9ce3f39b8d0ae034215be19c8688bdb285aa8a), though, it's not tagged, can we use that?

Reply:
Though for this one, I would recommend using the latest commit all the time, if there is not any issues. We can use `unstableGitUpdater` - https://github.com/NixOS/nixpkgs/blob/03c07e5a7a08a24d24bd0d0d79d0b98d3e657c18/pkgs/common-updater/unstable-updater.nix to easily update it ourselves or bots bump this